.github/ISSUE_TEMPLATE/regular-issue.yaml

```diff
@@ -0,0 +1,47 @@
+name: Generic Issue
+description: Open a generic Issue for the OpenJS CPC
+labels:
+  - cross-project-council-agenda
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ### OpenJS Cross Project Council
+
+        Thank you for opening an issue for the OpenJS Cross Project Council.
+
+        **Please remember to follow the Code of Conduct and be respectful.**
+  - type: input
+    id: topic
+    attributes:
+      label: What is this Issue about?
+      description: How can the CPC help you out?
+    validations:
+      required: false
+  - type: textarea
+    id: details
+    attributes:
+      label: Request Details
+      description: Provide us more details about your request to the CPC
+    validations:
+      required: true
+  - type: dropdown
+    id: type
+    attributes:
+      label: Request Type
+      description: What is this Issue about?
+      options:
+        - General Question
+        - Resource Request
```

Comment:
Resource requests belong with operations (there's an email address for those).

Reply:
Same as my previous comment. How are people supposed to know where to go for these sort of requests. Do we have some docs anywhere? 👀